scripts/publish-packages.sh

```
@@ -61,9 +61,9 @@ publish_if_needed() {
     fi
     
     echo "📦 Publishing $package_name@$version..."
-    npm install
-    npm run build
-    npm publish --otp=$NPM_OTP
+    pnpm install
+    pnpm run build
+    pnpm publish --otp=$NPM_OTP
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)